src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js

```diff
@@ -61,11 +61,13 @@ const AddProposalPanel = React.memo(({ setProposalData }) => {
 
   const fundingMode = formData.proposalType === FUNDING_PROPOSAL
 
-  const forumRegex = new RegExp(connectedGarden.forumURL)
```

Comment:
Note that this was the `forumURL` because Gardens can specify their own forum. So we need to include a regEx that uses the `forumURL` as variable.

Reply:
Right. Idk how it was working without escape the string, it will just blow up because the double slash on https:// it's nots escaped in orignal forumURL.

Anyway, I'll add them back mergeing with the discord